[Bulk] Validation: Fixed issues in ODatadocs (#219)
MicrosoftDocs/OData-docs: Odata-docs/connectedservice/generating-code-for-vb-clients.md

---
title: "Generating code for VB clients"

> description: OData Connected Service allows you to generate client code both for C# and Visual Basic.

Reviewer: ```suggestion
description: Describes how to generate client code for Visual Basic.
```
